physionet-django/console/templates/console/user_management.html

```diff
@@ -111,21 +111,21 @@ <h3>Profile</h3>
     </div>
   {% endfor %}
 
-  {% if groups %}
-      <div class="row mb-1">
-        <div class="col-md-3">
-          User Groups:
-        </div>
-        <div class="col-md-9">
-          {% for group in groups %}
-              <a href="{% url 'user_group' group.name %}">{{ group.name }}</a>
-          {% empty %}
-            N/A
-          {% endfor %}
-        </div>
+  <br>
+  <h3>Permission Groups</h3>
+  <hr>
+    <div class="row mb-1">
+      <div class="col-md-6">
+        The user is a member of the following groups:
       </div>
-
-  {% endif %}
+      <div class="col-md-9">
+        {% for group in groups %}
+            <ul><li><a href="{% url 'user_group' group.name %}">{{ group.name }}</a></li></ul>
```

Comment:
`<ul>` should go above `{% for group in groups %}` and `</ul>` should go below `{% endfor %}`. Otherwise this creates a new list each loop (rather than a new item in the list).